presidio-analyzer/setup.py

```diff
@@ -11,8 +11,11 @@
 with open(path.join(this_directory, 'README.MD'), encoding='utf-8') as f:
     long_description = f.read()
 
-with open(os.path.join(parent_directory, "VERSION")) as version_file:
-    __version__ = version_file.read().strip()
+try:
+    with open(os.path.join(parent_directory, "VERSION")) as version_file:
+        __version__ = version_file.read().strip()
+except Exception:
+    __version__ = os.environ.get("PRESIDIO_VERSION", "0.0.1-alpha")
```

Comment:
Do we want to inject this env variable during e2e testing or use the default 0.0.1-alpha?

Reply:
This should be added to the pipeline settings, and will be read here